docs/source/quick-start.mdx

```diff
@@ -62,26 +62,34 @@ Once you have your User Access Token, run the following command in your terminal
 huggingface-cli login
 ```
 
-Or if you prefer to work from a Jupyter or Colaboratory notebook, then use [`login`]:
+Alternatively, you can programmatically login using [`login`] in a notebook or a script:
 
 ```py
 >>> from huggingface_hub import login
 >>> login()
 ```
 
 <Tip>
-
-You can also provide your token to the functions and methods. This way you don't need to
-store your token anywhere.
-
+It is also possible to login programmatically without asking the user for the token by directly
+passing the token to [`login`] (i.e. `login(token="hf_xxx")`). If you do so, be careful when
+sharing your source code. It is a best practice to load the token from a secure vault instead
+of saving it in plain in your codebase/notebook.
```

Comment:
You'll need to add a blank line after `<Tip>` so it renders properly. I would also merge this tip with all the ones below it, so you don't have four separate tips. Tips are kind of like side content, but in this case, I think all of this content is quite important!

```suggestion
It is also possible to login programmatically without being prompted to enter your token by directly
passing the token to [`login`] like `login(token="hf_xxx")`. If you do so, be careful when
sharing your source code. It is a best practice to load the token from a secure vault instead
of saving it explicitly in your codebase/notebook.
```